tests/conftest.py

```diff
@@ -297,6 +297,10 @@ def app_config(app_config, mock_datacite_client):
     app_config["USERS_RESOURCES_SERVICE_SCHEMA"] = NotificationsUserSchema
 
     app_config["RDM_RESOURCE_ACCESS_TOKENS_ENABLED"] = True
+
+    # Users are verified by default. This will disable the automatic creation of moderation requests after publishing a record.
+    # When testing unverified users, there is a "unverified_user" fixture for that purpose.
+    app_config["ACCOUNTS_DEFAULT_VERIFIED_STATUS"] = True
```

Comment:
This simplifies testing. 

That way, after `publish` there are no moderation requests created in tests and it's easier to control which user is verified / not-verified (I created explicit users for that purpose).